app/domain/queries/find_content.rb

```diff
@@ -38,12 +38,21 @@ def order_by
   def initialize(filter)
     @organisation_id = filter.fetch(:organisation_id)
     @document_type = filter.fetch(:document_type)
-    @search_term = filter[:search_term]
+    @search_term = parse_search_term(filter[:search_term]) if filter[:search_term]
     @page = filter[:page] || 1
     @page_size = filter[:page_size] || DEFAULT_PAGE_SIZE
     @date_range = filter.fetch(:date_range)
   end
 
+  def parse_search_term(search_term)
+    protocol = /http(s)?:\/\//
+    domain = /(www\.)?gov.uk/
```

Comment:
Think you need to escape the second `.`
```suggestion
    domain = /(www\.)?gov\.uk/
```